app/views/devise/passwords/new.html.erb

```diff
@@ -1,16 +1,26 @@
-<h2>Forgot your password?</h2>
+<div class="container">
+  <div class="d-flex justify-content-center h-100">
+    <div class="card forgot_password">
+      <div class="card-header">
+				<h3>Forgot your password?</h3>
+			</div>
+      <div class="card-body remember">
+        <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+          <%= render "devise/shared/error_messages", resource: resource%>
 
-<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
+          <div class="input-group form-group">
+            <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control", placeholder: "Email" %>
+          </div>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+          <div class="form-group">
+            <%= f.submit "Send", class: "btn float-right btn-default btnbackground" %>
+            <br /><%= f.label :Send_me_reset_password_instructions, class: "remember"%>
+          </div>
+        <% end %>
+      </div>
+      <div class="card-footer">
+				<%= render "devise/shared/links"%>
+			</div>
```

Comment:
Indentation issue

Reply:
I have checked all files that you mentioned but there is no indentation issue in original files.